### DISCUSSION TITLE
Using the last-selected note in Keyboard mode for each new step added in Sequencer mode, just like Pocket Operator does

### DISCUSSION THREAD

**Discussion by ajsoriar** [bangcorrupt/hacktribe#289] [Ideas]
Hi! 
I am new here.
I'm considering installing Hacktrive. I am very disappointed with the device. Sampler version in my case.

I would like to know if Hacktribe includes improvements to the sequencer mode for adding notes.
One of the most frustrating aspects for me is the sequencer and the workflow for adding notes and creating melodies. 
It would be great if the last-selected note in Keyboard mode could be automatically used when activating steps in the Sequencer mode, much like how a Pocket Operator works. So, when returning to Keyboard mode, we can select a new note, and back to Sequencer mode, we can easily add new steps with the last-selected note. 

Is there a feature like this available or any equivalent improvements in the sequencer?

In any case nice to know that somebody is trying to fix this half-finished device. Thanks, bangcorrupt!

**Comment by bangcorrupt** [comment]
This isn't a feature yet, but I think I can probably add it at some point.  It could also be useful in trigger mode, as discussed in #159.
